engine/table/src/main/java/io/deephaven/engine/table/impl/TableDefaults.java

```diff
@@ -307,7 +307,7 @@ default Table renameAllColumns(UnaryOperator<String> renameFunction) {
     @ConcurrentMethod
     @FinalDefault
     default Table formatRowWhere(String condition, String formula) {
-        return formatColumnWhere(ColumnFormattingValues.ROW_FORMAT_NAME, condition, formula);
+        return formatColumnWhere("*", condition, formula);
```

Comment:
This seems less clear than a shared constant.

Reply:
1. `"*"` seems to be the wildcard we expect users to enter manually, if they use `formatColumns` instead of `formatRowWhere`
2. The previous version of the code was mapping the constant to the wildcard a layer deeper, and I (think) I was trying to standardize a bit.
3. I've exposed and used a constant for this value, as the "user-visible" version.